From 60f90ec3cc3e417c4b8fe2ea25ddf6e5ff99bde0 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 14 Dec 2020 14:51:36 +0800
Subject: [PATCH] projector: fix cannot find module 'd3'

[snip]
$ bazel run tensorboard/plugins/projector/vz_projector:standalone
tensorboard/plugins/projector/vz_projector/projectorScatterPlotAdapter.ts:16:21 - error TS2307: Cannot find module 'd3'.

16 import * as d3 from 'd3';
[snip]

Upstream-Status: Submitted [https://github.com/tensorflow/tensorboard/pull/4465]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorboard/plugins/projector/vz_projector/BUILD | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorboard/plugins/projector/vz_projector/BUILD b/tensorboard/plugins/projector/vz_projector/BUILD
index f173cd78..1529440d 100644
--- a/tensorboard/plugins/projector/vz_projector/BUILD
+++ b/tensorboard/plugins/projector/vz_projector/BUILD
@@ -63,6 +63,7 @@ tf_ts_library(
         "//tensorboard/webapp/third_party:tfjs",
         "@npm//@polymer/decorators",
         "@npm//@polymer/polymer",
+        "@npm//@types/d3",
         "@npm//d3",
         "@npm//numeric",
         "@npm//three",
-- 
2.18.2

